service_user.go

```diff
@@ -78,6 +86,30 @@ func (h *ServiceUsersHandler) Get(project, serviceName, username string) (*Servi
 	return nil, err
 }
 
+// Update modifies the given Service User in Aiven.
+func (h *ServiceUsersHandler) Update(project, service, username string, update ModifyServiceUserRequest) (*ServiceUser, error) {
+	var DefaultOperation = "reset-credentials"
+	if update.Operation == nil {
+		update.Operation = &DefaultOperation
+	}
+	path := buildPath("project", project, "service", service, "user", username)
+	svc, err := h.client.doPutRequest(path, update)
+	if err != nil {
+		return nil, err
+	}
+	var r ServiceResponse
+	errR := checkAPIResponse(svc, &r)
+	if errR == nil {
+		for _, user := range r.Service.Users {
+			if user.Username == username {
+				return user, nil
+			}
+		}
+		return nil, errors.New("User not found")
```

Comment:
An error string shall neither be capitalized nor end with punctuation according to Go standards.